packages/zowe-explorer/src/shared/utils.ts

```diff
@@ -375,9 +379,13 @@ export async function compareFileContent(
     label?: string,
     profile?: imperative.IProfileLoaded
 ): Promise<void> {
+    node = node ?? TreeProviders.ds.openFiles?.[doc.uri.fsPath] ?? TreeProviders.uss.openFiles?.[doc.uri.fsPath];
```

Comment:
Curious if we should also search for the node in the `uploadContent` method before we check for `== null`.

Reply:
At the time `uploadContent` is called, we should have already handled the case where node is null or undefined in `saveFile` for data sets or `saveUSSFile` for USS files.